exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/VarLenAbstractEntryReader.java

```diff
@@ -0,0 +1,266 @@
+/*******************************************************************************
```

Comment:
Please license headers, Travis fails. Consider other classes as well.

Reply:
I have fixed the headers and modified my maven script to always set "-Dlicense.skip" to true; shouldn't be set by default?

Reply:
It should be disabled by default, IMO. There is no need to check license headers all the time. Please either run license check explicitly or rely on Travis.